packages/volto/news/4819.feature

```diff
@@ -0,0 +1 @@
+Add setting ``unwantedControlPanelsFields`` and use it in the function ``filterControlPanelsSchema``. @wesleybl
```

Comment:
FYI, MyST uses single backticks for inline literals, whereas reStructuredText uses double backticks.

```suggestion
Add setting `unwantedControlPanelsFields` and use it in the function `filterControlPanelsSchema`. @wesleybl
```

Reply:
> FYI, MyST uses single backticks for inline literals, whereas reStructuredText uses double backticks.

Thanks for the info! Done.